packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-## 41.6.0 - October 25, 2017
+## 41.8.0 - Novemeber 16, 2017
```

Comment:
Let's use Thursdays since that's when we release.

Reply:
Thanks for fixing the date of last release. There's a typo in Novemeber.